test/tap/umask.js

```diff
@@ -0,0 +1,64 @@
+var path = require("path")
+var test = require("tap").test
+var rimraf = require("rimraf")
+var mkdirp = require("mkdirp")
+var common = require("../common-tap.js")
+var npmlog = require("npmlog")
```

Comment:
This isn't used in the test, but you should be setting the log level to `silent` -- lotta warnings getting printed out right now.